server/index.js

```diff
@@ -4,6 +4,8 @@ import { config } from 'dotenv';
 import logger from 'morgan';
 import validator from 'express-validator';
 import cors from 'cors';
+import swaggerUI from 'swagger-ui-express';
+import swaggerDocument from '../swagger.json';
```

Comment:
I think it's better to move the swagger document to the server folder.

Reply:
since its just the server that's in the server file, I think it should be outside with the other files. Or why do you think it should be inside?

Reply:
> I think it's better to move the swagger document to the server folder.

since its just the server that's in the server file, I think it should be outside with the other files. Or why do you think it should be inside?